### DISCUSSION TITLE
Session expires? (Periodic access)

### DISCUSSION THREAD

**Discussion by pgi-jsanchez** [requarks/wiki#6395] [Error / Bug Report]
Hello.

A time ago I participated in this discussion https://github.com/requarks/wiki/discussions/6287#discussioncomment-5447356 about this topic, but due the question is marked as answered, I still doubt if this is the expected behavior of if this is a bug that I still experience and requires a second look about it or if there's something misconfigured.

After certain time of login, even if I am editing a page, the session expires or unable me to upload resources unxecpectedly, my workarround to fix it is with /logout request, but it's uncomfortable to do that everytime you want to fix this or you open the site after certain time and the session shows exists but certain functions does not work.

Steps to reproduce: 
1. Open wikijs for the first time and login
2. Use the site during a period of time or close the page without login
3. Access a page and check if the create new page or edit button are present or open the site at the next day

Current behavior:
Certain functions does not show, relogin without using /logout it's looped because the session is not updated and does not login properly.

Expected behavior:
Logout when session expires when page is not guest accesible, avoid login loop and allow features according it's user.

Environment:
Google Chrome version: 112.0.5615.165
WikiJS version: 2.5.298

**Comment by Leinad4Mind** [comment]
Getting the same issue, the page opens but login doesnt work, I get a blank page because it loops.

/logout for me doesnt solves it. :/
